frontend/package.json

```diff
@@ -3,13 +3,14 @@
   "private": true,
   "scripts": {
     "build": "turbo build",
+    "typecheck": "turbo typecheck",
     "dev": "turbo dev",
-    "lint": "turbo lint",
+    "lint": "turbo lint typecheck",
     "format": "prettier --write \"**/*.{ts,tsx,md}\""
   },
   "devDependencies": {
     "@storybook/test-runner": "^0.19.1",
-    "prettier": "^3.2.5",
+    "prettier": "3.3.3",
```

Comment:
We want this version locked?

Reply:
Per the [prettier doc](https://prettier.io/docs/en/install.html), it appears to be the recommended approach so I went with an exact version.

> Install an exact version of Prettier locally in your project. This makes sure that everyone in the project gets the exact same version of Prettier. Even a patch release of Prettier can result in slightly different formatting, so you wouldn’t want different team members using different versions and formatting each other’s changes back and forth.